sdk/communication/azure-communication-callautomation/tests/test_call_media_client.py

```diff
@@ -118,6 +139,27 @@ def test_play_file_to_all(self):
         self.assertEqual(expected_play_request.play_sources[0].play_source_cache_id, actual_play_request.play_sources[0].play_source_cache_id)
         self.assertEqual(expected_play_request.play_to, actual_play_request.play_to)
         self.assertEqual(expected_play_request.play_options, actual_play_request.play_options)
+    
+    def test_play_file_to_all_with_barge_in(self):
+        mock_play = Mock()
+        self.call_media_operations.play = mock_play
+        play_source = FileSource(url=self.url)
+
+        self.call_connection_client.play_media(play_source=play_source, interrupt_call_media_operation=True)
```

Comment:
Could you please add a test validating when `interrupt_call_media_operation` is not passed that it is correctly defaulted to `False`?

Reply:
As i mentioned above, keeping interrupt media flag as bool/none to match the generated model.